vite.config.js

```diff
@@ -39,6 +39,11 @@ export default defineConfig(({ mode }) => {
       proxy: {
         "/_matrix": env.VITE_DEFAULT_HOMESERVER || "http://localhost:8008",
       },
+      strictPort: true,
```

Comment:
Oh, this was accidentally committed and was me trying to get vite's websocket working when loading it through an ngrok tunnel, otherwise it just reloaded the page after timing out the websocket connection.